src/libs/PolicyUtils.ts

```diff
@@ -131,8 +131,8 @@ function isExpensifyTeam(email: string | undefined): boolean {
 /**
  * Checks if the current user is an admin of the policy.
  */
-const isPolicyAdmin = (policy: OnyxEntry<Policy> | EmptyObject): boolean => policy?.role === CONST.POLICY.ROLE.ADMIN;
-
+const isPolicyAdmin = (policy: OnyxEntry<Policy> | EmptyObject): boolean =>
+    policy?.role === CONST.POLICY.ROLE.ADMIN || policy?.employeeList?.[policy?.owner]?.role === CONST.POLICY.ROLE.ADMIN;
```

Comment:
I think we should check the role of the **_current user accountID_**  here. Or it'll introduce regression - non-admin members can also see protected items.


https://github.com/Expensify/App/assets/117511920/4925ecfb-43d7-4018-b210-b9cd9356e15e